### PR TITLE
BaSP-TG-83: Created Admin form, added post and edit features, added route on Layout component

### PR DESCRIPTION
**Pull request by marcomigliaro**
- Add route on Layout Component
- Create Form
- Add feature
- On editing, form should contain previous info
- Edit feature
- Success message on Add and Edit

### CODE REVIEW

File: src/Components/Admins/AdminForm/AdminForm.jsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+      lastName: '',
+      email: '',
+      password: '',
+      active: ''
+    });
+  };
+
+  return (
+    <div className={styles.container}>
+      <h2 className={styles.h2}>Add New Admin</h2>
```
Comment (cocolobos) [inline src/Components/Admins/AdminForm/AdminForm.jsx:47]: the class name should be different from the tag name

File: src/Components/Admins/AdminForm/AdminForm.jsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+          <div className={styles.active}>
+            <label>Active</label>
+            <input type="text" name="active" value={adminInput.active} onChange={onChange} />
+          </div>
+          <button type="submit" className={styles.confirmBtn}>
+            Confirm
+          </button>
+        </div>
+      </form>
+      <a href="/admins" className={styles.addBtn}>
```
Comment (cocolobos) [inline src/Components/Admins/AdminForm/AdminForm.jsx:79]: this should be a button an have an onclick function that redirects

File: src/Components/Admins/AdminForm/AdminFormEdit.jsx [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+            <input type="text" name="active" value={adminInput.active} onChange={onChange}></input>
+          </div>
+          <div>
+            <button type="submit" className={styles.confirmBtn}>
+              Confirm
+            </button>
+          </div>
+        </div>
+      </form>
+      <a href="/admins" className={styles.addBtn}>
```
Comment (cocolobos) [inline src/Components/Admins/AdminForm/AdminFormEdit.jsx:78]: this should be a button an have an onclick function that redirects

File: src/Components/Admins/index.js [hunk 4]
```diff
@@ -1,11 +1,43 @@
...
+    setList(list.filter((listItem) => listItem._id !== _id));
+  };
 
-function Admins() {
   return (
     <section className={styles.container}>
-      <h2>Admins</h2>
+      <h2 className={styles.h2}>Admins</h2>
+      <div>
+        <a href="/admins/add" className={styles.addBtn}>
```
Comment (cocolobos) [inline src/Components/Admins/index.js:34]: this should be a button an have an onclick function that redirects

**Review by cocolobos — COMMENTED** [review]
Edit functionality not working

**Comment by cocolobos** [comment]
After adding or editing not alert displayed, and also should redirect to table

File: src/Components/Admins/AdminForm/AdminForm.jsx [hunk 5]
```diff
@@ -1,13 +1,90 @@
...
+    });
+  };
+
   return (
-    <div>
-      <div>Form</div>
-      <a href="/admins" className={styles.addBtn}>
+    <div className={styles.container}>
+      <h2 className={styles.title}>Add New Admin</h2>
+      <form onSubmit={onSubmit} className={styles.form}>
```
Comment (cocolobos) [inline src/Components/Admins/AdminForm/AdminForm.jsx:54]: don't use tag name for classes

File: src/Components/Admins/Modal/AdminFormEdit.jsx [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+
+  if (!showModal) {
+    return null;
+  }
+
+  return (
+    <div className={styles.modalContainer}>
+      <div className={styles.editModal}>
+        <h2 className={styles.title}>Edit Admin</h2>
+        <form onSubmit={handleSubmit} className={styles.form}>
```
Comment (cocolobos) [inline src/Components/Admins/Modal/AdminFormEdit.jsx:46]: same here

**Review by cocolobos — APPROVED** [review]

**Review by JulianDeme — APPROVED** [review]

**Pull request merged**